src/main/drivers/compass/compass_spi_hmc5883l.c

```diff
@@ -21,17 +21,22 @@
 
 #include <platform.h>
 
+#include "config/parameter_group.h"
+
 #include "drivers/io.h"
 #include "drivers/bus_spi.h"
 
 #include "compass.h"
 #include "compass_hmc5883l.h"
 
+#define SPI_CFG_TO_DEV(x) ((x) - 1)
```

Comment:
This name is confusing
Function will be better, and it will be much cleaner if spiDevice is passed around as pointer to device record instead of enum